helm/kubeless/templates/zk-stateful.yaml

```diff
@@ -20,6 +20,8 @@ spec:
           env:
             - name: ZOO_SERVERS
               value: server.1=zoo-0.zoo:2888:3888:participant
+            - name: ALLOW_ANONYMOUS_LOGIN
```

Comment:
the chart could support a more secure setup by randomly generating a password, something to improve upon in the future